protocol/0042-LIQF-setting_fees_and_rewarding_lps.md

```diff
@@ -146,6 +146,33 @@ An existing LP has `average entry valuation 1090.9` and `S=110`. Currently the s
 (average entry valuation) = 1090.9
 ```
 
+### Calculating supplied liquidity from probability of trading 
+
+At every vega time change calculate the supplied liquidity provided by each committed LP. 
+This is done by taking into account all the volume they're providing between the tightest price monitoring bound and then 
```

Comment:
what will happen if max_lp_vol_price < tightest price monitoring upper bound

Reply:
That's the expected case: 
```
lower bound from tightest price mon < min_lp_vol_price < mid < max_lp_vol_price < tightest price monitoring upper bound
```